MAUI/Scheduler/header.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Header in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about to customize header height, date format and appearance of Syncfusion .NET MAUI Scheduler (SfScheduler) control and more. 
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Header in .NET MAUI Scheduler (SfScheduler)
+
+You can learn about SfScheduler's header height, date format, and appearance support of the SfScheduler.
+
+## Customize header height in scheduler
+
+You can customize the default `Height` property of the `HeaderView` in the `SfScheduler.` By default, the header height is `50.`
+
+{% tabs %}  
+{% highlight xaml %}
+
+<scheduler:SfScheduler x:Name="Scheduler" 
+                       View="Week">
+    <scheduler:SfScheduler.HeaderView>
+        <scheduler:SchedulerHeaderView Height="100">
+        </scheduler:SchedulerHeaderView>
+    </scheduler:SfScheduler.HeaderView>
+</scheduler:SfScheduler>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.Scheduler.View = SchedulerView.Week;
+this.Scheduler.HeaderView.Height = 100;
+
+{% endhighlight %}  
+{% endtabs %}
+
+## Header date format
+
+The header date format can be customized by using the `TextFormat` property of the `SfScheduler.` By default, the header text format is `MMMM yyyy.`
+
+{% tabs %}  
+{% highlight xaml %}
+
+<scheduler:SfScheduler x:Name="Scheduler" 
+                       View="Week">
+    <scheduler:SfScheduler.HeaderView>
+        <scheduler:SchedulerHeaderView TextFormat="MMM yy">
+        </scheduler:SchedulerHeaderView>
+    </scheduler:SfScheduler.HeaderView>
+</scheduler:SfScheduler>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.Scheduler.View = SchedulerView.Week;
+this.Scheduler.HeaderView.TextFormat = "MMM yy";
+
+{% endhighlight %}
+{% endtabs %}
+
+## Header customization
+
+The header can be customized by using the `HeaderView` property of the `SfScheduler.`
+
+### Customize header appearance using text style
+
+You can style the background color, and textStyle by using the properties such as `Background,` and `TextStyle` properties of `HeaderView.`
```

Comment:
You can style the background color and textStyle by using the properties such as `Background` and `TextStyle` properties of `HeaderView.`